cranelift/filetests/filetests/isa/x64/conditional-values.clif

```diff
@@ -10,7 +10,7 @@ block0(v0: i8, v1: i32, v2: i32):
 ;   pushq   %rbp
 ;   movq    %rsp, %rbp
 ; block0:
-;   testb   $1, %dil
+;   testb   %dil, %dil
```

Comment:
Good catch! Just to make sure, do we have a runtest somewhere that tests with a nonzero-but-even value to catch LSB-only checks like this?

Reply:
We do in the runtests: there's a case that checks the return value with `2` as the argument to `select` in `runtests/br.clif`.